ci/Dockerfile-envoy

```diff
@@ -31,9 +31,16 @@ RUN adduser --group --system envoy
 
 RUN mkdir -p /etc/envoy
 
-ADD build_release_stripped/envoy /usr/local/bin/envoy
+ADD arm64/build_release_stripped/envoy /usr/local/bin/envoy-arm64
```

Comment:
Effectively you put everything in the image here, i.e. making the image big, doing `rm` later doesn't remove the binary form docker layers. Can this actually just be a build-arg?

Reply:
Has updated. Please check it.